### PR TITLE
[dotnet][tvos] Enable ICU (instead of using Invariant) and additional tests

### PR DESCRIPTION
**Pull request by spouliot**
note: link all should fail due to https://github.com/xamarin/xamarin-macios/issues/11243

### CODE REVIEW

File: tests/xharness/Harness.cs [hunk 1]
```diff
@@ -413,21 +413,25 @@ void AutoConfigureIOS ()
...
 			IOSTestProjects.Add (new iOSTestProject (Path.GetFullPath (Path.Combine (RootDirectory, "linker", "ios", "link sdk", "link sdk.csproj"))) {
 				Configurations = new string [] { "Debug", "Release" },
 				IgnoreMacCatalystVariation = false,
 			});
 			IOSTestProjects.Add (new iOSTestProject (Path.GetFullPath (Path.Combine (RootDirectory, "linker", "ios", "link sdk", "dotnet", "iOS", "link sdk.csproj"))) { Configurations = new string [] { "Debug", "Release" }, IsDotNetProject = true, SkipiOSVariation = false, SkiptvOSVariation = true, SkipwatchOSVariation = true, SkipTodayExtensionVariation = true, SkipDeviceVariations = false, SkipiOS32Variation = true });
+			IOSTestProjects.Add (new iOSTestProject (Path.GetFullPath (Path.Combine (RootDirectory, "linker", "ios", "link sdk", "dotnet", "tvOS", "link sdk.csproj"))) { Configurations = new string [] { "Debug", "Release" }, IsDotNetProject = true, SkipiOSVariation = true, SkiptvOSVariation = true, SkipwatchOSVariation = true, SkipTodayExtensionVariation = true, SkipDeviceVariations = false, SkipiOS32Variation = true, GenerateVariations = false, TestPlatform = TestPlatform.tvOS, });
```
Comment (mandel-macaque) [inline tests/xharness/Harness.cs:434]: I can think something to make use write less next time :/

**Review by mandel-macaque — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Review by rolfbjarne — COMMENTED** [review]
Did you forget to add the new .NET csprojs for tvOS?

**Comment by spouliot** [comment]
> Did you forget to add the new .NET csprojs for tvOS?

Of course 😄

**Review by rolfbjarne — APPROVED** [review]

**Pull request merged**